src/caustics/lenses/multiplane.py

```diff
@@ -188,6 +190,7 @@ def raytrace(
         -------
         (Tensor, Tensor)
             The reduced deflection angle.
+        *Unit: radians*
```

Comment:
The spacing doesn't match the rest of the usages.

Reply:
Updated, maintaining consistency now.